charts/agent-operator/templates/_helpers.tpl

```diff
@@ -36,10 +36,15 @@ Common labels
 {{- define "ga-operator.labels" -}}
 {{ include "ga-operator.selectorLabels" . }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/component: operator
+app.kubernetes.io/part-of: {{ template "ga-operator.name" . }}
```

Comment:
moving the `component` label makes sense, thanks for catching that! if possible, i would omit the `part-of` label - if users need this, they can now create this label with your other `customLabels` addition.